core/math/geometry.cpp

```diff
@@ -1211,7 +1211,9 @@ Vector<Vector<Point2> > Geometry::_polypath_offset(const Vector<Point2> &p_polyp
 		case END_SQUARE: et = etOpenSquare; break;
 		case END_ROUND: et = etOpenRound; break;
 	}
-	ClipperOffset co;
+	// Miter limit is specified in multiples of p_delta, 
+	// so there's no need to scale it before offseting
+	ClipperOffset co(p_miter_limit, p_arc_tolerance * SCALE_FACTOR);
```

Comment:
Fixed arc tolerance to be multiplied by `SCALE_FACTOR`. Before it would introduce very small tolerance so that changes to this parameter were negligible, hence significant performance drop. Now the performance is mostly the same compared to other types of offsetting (`SQUARE`, `MITER`).

So since it's useful now, shall we remain this parameter or drop it, WDYT? If we decide to drop it, the arc_tolerance fix must be applied anyway:

```suggestion
	ClipperOffset co(p_miter_limit, 0.25 * SCALE_FACTOR); // default value from ClipperOffset
```
Otherwise shall be good to merge.